landscape/categories.md

```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
   - _Secret Management_
     - Tools that help to manage, encrypt or distribute secrets and other sensitive information.
 
-- **Identity & Access Control**
-  - _Identity_
-    - SPIFFE, identity providers, OpenID, LDAP, Okta
-  - _Access Controls_
+- **Identity Lifecycle and & Access Management**
+  - _Identity Provisioning and Lifecycle Management_
+    - Enrollment and rovisioning of identities for individuals
+      - Lifecycle management of identities and associated roles and entitlements
+      - Includes implementations of protocols like SCIM for cross-domain provisioning
+      - See, e.g. NIST SP 800-63A
+    - Identity provisioning for service and machine identities
+      - Cloud-native identity for ephemeral workloads, e.g. the spiffe project
+      - Historical implementations may be based on, or need to interact with, systems like Kerberos
+    - Directory and attribute management
+      - Typically implemented atop protocols like LDAP, or other hierarchical and graph-based stores
+  - _Identity Consumption and Single-sign-on_
+    - Identity federation and single sign-on
+      - Typically supporting OpenID connect and SAML
+    - Anonymization
+    - Identity intelligence
+      - Know-Your-Customer/Anti-Money-Laundering projects
+      - Anti-fraud systems
+  - _Access control and enforcement engines_
+    - Attribute and policy-based resource authorization engines and libraries
+      - Including general policy engines
+    - Role-based access control engines
+      - Including libraries and standards for common roles
+    - Privileged Access Management
+      - Applying higher levels of control for sensitive actions, including just-in-time access and enhanced logging
+  - _Resource-specific access layers_
```

Comment:
Copying comment over: Could you clarify what you mean by layer? Layer implies some sort of stack.

Seems like we could replace this with Resource-specific access control?